hack/addons/kustomize/cluster-autoscaler/manifests/helm-values.yaml

```
@@ -20,7 +21,7 @@ tolerations:
 
 # Limit a single cluster-autoscaler Deployment to a single Cluster.
 autoDiscovery:
-  clusterName: tmpl-clustername-tmpl
+  clusterName: tmpl-clusteruuid-tmpl
```

This shouldn't be changing.
```suggestion
  clusterName: tmpl-clustername-tmpl
```